client/a8c-for-agencies/components/site-configurations-modal/index.tsx

```diff
@@ -205,34 +213,41 @@ export default function SiteConfigurationsModal( {
 							onChange={ toggleAllowClientsToUseSiteHelpCenter }
 							checked={ allowClientsToUseSiteHelpCenter }
 							name="is_fully_managed_agency_site"
-							disabled={ isSubmitting }
+							disabled={ isDevSite }
```

Comment:
Maybe I missed something, but shouldn't we keep the `CheckboxControl` component disabled even when the form is being submitted?

```suggestion
							disabled={ isDevSite || isSubmitting }
```

Reply:
Oh, you are right here. Thank you for catching.

Reply:
Thank you! 🙂